tests/integration/test_trilu_indices.py

```diff
@@ -192,23 +192,23 @@ class TestTriluIndicesFromErrors:
     @pytest.mark.parametrize(
         "dimension", (1, 3), ids=lambda dimension: f"(dim={dimension})"
     )
-    def test_arr_non_2d(self, dimension, size):
+    @pytest.mark.parametrize("func", FUNCTIONS_INDICES_FROM)
+    def test_arr_non_2d(self, func, dimension, size):
+        expected_exc = ValueError
         shape = size * dimension
         a = num.ones(shape, dtype=int)
-        msg = "input array must be 2-d"
-        with pytest.raises(ValueError, match=msg):
-            num.tril_indices_from(a)
-
-    def test_arr_0d(self):
-        a = num.array(3)
-        msg = "input array must be 2-d"
-        with pytest.raises(ValueError, match=msg):
-            num.tril_indices_from(a)
```

Comment:
What's the reason for deleting this test? (This case doesn't seem to be covered anywhere else)

Reply:
It was an unfortunate accident, adding it back with the intended changes.